test/common/router/config_impl_speed_test.cc

```diff
@@ -27,19 +27,21 @@ using testing::ReturnRef;
  * Generates a request with the path:
  * - /shelves/shelf_x/route_x
  */
-static Http::TestRequestHeaderMapImpl genRequestHeaders(int route_num) {
+static Http::TestRequestHeaderMapImpl genRequestHeaders(int route_num,
+                                      std::string req_path_header) {
```

Comment:
`req_path_prefix`?

Reply:
Also, this should be `absl::string_view` or if not possible `const std::string&`.

Reply:
ACK